contrib/inventory_builder/requirements.txt

```diff
@@ -1,3 +1,4 @@
 configparser>=3.3.0
 ruamel.yaml>=0.15.88
 ipaddress
+boto3
```

Comment:
This requirements.txt file seems for contrib/inventory_builder/inventory.py according to the first commit 7b86b87dcac0c7ff459971edc43018b570c0625d.
How about adding contrib/aws_inventory/requirements.txt for kubespray-aws-inventory.py on the same path?

Reply:
creating a new one will don't take much , but two different files and 2 different time installation for similar purpose.

Reply:
let discuss it with other , see which approach is acceptable to majority.

Reply:
I don't really like having 2 files but as we don't change this often and I also don't like installing things non necessary... I'd say that a requirements-aws is kind of cleaner

Reply:
Ok, Then "requirements-aws.txt" in "contrib/aws_inventory/" sounds good to all. Please confirm.

Reply:
requirements.txt is a common file name in Python without `-aws` I feel.

Reply:
Indeed, sorry my comment was misleading 😄 I'm all in for a "requirement-aws-like" but with respect to convention => requirement.txt in aws folder

Reply:
done as required.